### ISSUE TITLE
[Usable - Bot framework - Chat window]: 'Keyboard controls' screen doesn't appear by default on the screen.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4713]
**User Experience:**
Visual users and people having limited cognitive abilities get affected as they miss knowing the useful information regarding the keyboard controls text. 

**Note:** 
User credentials should NOT be included in the bug.

**Pre-Requisite:**

1. Open environment URL https://compulim.github.io/webchat-loader/.
2. Select "Cdn.botframework.com/latest" list item under version combo box.
3. Select "Mock Bot" link and select "Open web chat in new window" link.

**Repro Steps:**

1. Open "Web chat" link and navigate to "Webchat" popup.
2. Navigate to the "Keyboard Controls" pane.
3. Verify whether the keyboard controls screen is visible by default or not.

**Actual Result:**
Keyboard controls screen doesn't appear on the screen by default. The screen appears only with focus on 'x' button. It disappears when focus moves away from 'x' button.

**Expected Result:**
Keyboard controls screen should appear on the screen by default.

**Reference Links:**

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fDNkZjhhMWU0NTA0YTQwM2MyNGRhMDhkYWY1NTkwMDNlfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwOTIwNTk4MzEzMTM2MjZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkzTXpZd09URTRNall3TXpzeE5qY3pOakE1TVRneU5qQXpPekU1T2pjMllUWXlNRE5pTnpZeFpqUmlObVE0T1dKbE0yRm1aVFZpTjJZd1pHSmhRSFJvY21WaFpDNTJNZz09fDBiZDAxMWU1M2M0YjQ0NGYyNGRhMDhkYWY1NTkwMDNlfGI0NDRlYzhjODQ4YzRiODI4Mzk1NmRjNDdlZjQ3NDky&sdata=dlJVM3lpMWQrRmtpc09HVTI0REh4NlVrU00yMG5xd0JlUFdBOHZuaHNOVT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-pgopinadhu%40microsoft.com&OR=Teams-HL&CT=1673616420039&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzAxMDUwOTQwMCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.

**Comment by pgopinadhu** [comment]
HCL; Accessibility; Benchmark; A11yUsable; A11ySev4; Feature-HCL-Botframeworkwebchat-Apr23; Win11-Edge(Chromium)

<img width="958" alt="Usable-botframework-keyboardcontrols-" src="https://user-images.githubusercontent.com/120019572/233826553-045dc21d-0b6c-47b6-8409-8cd645ea8db3.png">

**Comment by pgopinadhu** [comment]
<img width="958" alt="Usable-botframework-keyboardcontrols-" src="https://user-images.githubusercontent.com/120019572/234296953-862524a2-05c3-404f-a7cb-af044572943e.png">

**Comment by compulim** [comment]
This is by-design.

The keyboard help screen is designed to only show up when the user press <kbd>TAB</kbd> to put the focus in Web Chat.

Its design is similar to the ["Skip to content"](https://webaim.org/techniques/skipnav/) widget, which shows up on a webpage only when <kbd>TAB</kbd> into the page.

The design is also similar to what Power BI offers (see below).

<img width="468" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/1622400/73abf8fe-a89f-43c2-a779-dd3ab3c7bde6">

Recently, a few websites are trying to standardize on <kbd>SHIFT</kbd> + <kbd>/</kbd> to show the keyboard help screen. However, we cannot use this because it would interfere with text box.
